swap_meet/vendor.py

```diff
@@ -1,2 +1,79 @@
+from .item import Item
+
+# wave 1
 class Vendor:
-    pass
\ No newline at end of file
+    def __init__(self, inventory = None):
+        self.inventory = [] if inventory is None else inventory
+    
+    def add(self,item):
+        self.inventory.append(item)
+        return item
+    
+    def remove(self,item):
+        if item in self.inventory:
+            self.inventory.remove(item)
+            return item
+        return False
+    
+#wave 2
+    def get_by_id(self, id):
+        for item in self.inventory:
+            if item.id == id:
+                return item
+        else:
+            return None
```

Comment:
The `else` here is unnecessary and should be removed to reduce conditional complexity. 

if the condition on line 21 is never `True`, then the loop will exit and any remaining lines of code in the function will run. We don't need `else` to ensure `return None` will execute. 

```suggestion
            return None
```